core/src/mindustry/world/Block.java

```diff
@@ -375,6 +375,8 @@ public void setBars(){
         if(hasItems && configurable){
             bars.add("items", entity -> new Bar(() -> Core.bundle.format("bar.items", entity.items.total()), () -> Pal.items, () -> (float)entity.items.total() / itemCapacity));
         }
+        
+        if(flags.contains(BlockFlag.unitModifier)) stats.add(Stat.maxUnits, Strings.format("@ @", unitCapModifier < 0 ? '-' : '+', Math.abs(unitCapModifier)));
```

Comment:
This is a very simple case, you shouldn't need to use Strings.format here. Just concatenate.
I don't think the space between `+ num` is necessary.

Reply:
oh it kinda went wrong the first time and the numbers were in the 40-50 range, probably due to the char?

Reply:
Use double-quotes for string concatenation instead.